config/routes.rb

```diff
@@ -115,6 +115,10 @@
 
   get "/pages/*id" => "pages#show", as: :page, format: false
 
+  get "/:section/:file_name" => "posts#show",
+      constraints: ->(request) { request.params[:section].in?(%w[jobseeker-guides get-help-hiring]) },
```

Comment:
I think you can just do this:

```ruby
constraints: { section: %w[jobseeker-guides get-help-hiring] }
```

Reply:
It should also probably have a constraint on `file_name` (which might be better as a `post_name` as we don't really care that it's a file)?

e.g.:
```ruby
{ post_name: /[\w-]+/ }
```